### PR TITLE
feat(data-table): add select component to data-table, add zebra theme, add core html examples for momentum.design

### PR DESCRIPTION
**Pull request by daanishnasir**
- Select works within Data Table, added examples, zebra theme

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: core/scss/components/data-table/data-table.scss [hunk 1]
```diff
@@ -4,6 +4,22 @@
   .md-data-table {
     position: relative;
 
+    &.zebra {
```
Comment (bfbiggs) [inline core/scss/components/data-table/data-table.scss]: Wouldn't this be md-data-table--zebra? A standalone would make sense if we were using it in other places.

**Comment by mathsusan** [comment]
agree with Ben, love all the examples, I am wondering if the test cases need a bit more though

File: core/scss/components/data-table/data-table.scss [hunk 2]
```diff
@@ -4,6 +4,22 @@
   .md-data-table {
     position: relative;
 
+    &.md-data-table--zebra {
```
[the remark this answers is not in the document]
Comment (daanishnasir) [reply core/scss/components/data-table/data-table.scss]: updated

**Comment by daanishnasir** [comment]
Will be adding the site examples to this PR.

**Pull request merged**